src/scripts/main.js

```diff
@@ -1,3 +1,8 @@
 'use strict';
 
-// write code here
+const WALL = document.querySelector('.wall');
+const SPIDER = document.querySelector('.spider');
```

Comment:
```suggestion
const wall = document.querySelector('.wall');
const spider = document.querySelector('.spider');
```
it's not the constants